recipe/meta.yaml

```diff
@@ -21,23 +21,25 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
-    - ipython >=4.0.0
+    - python
+    - ipython >=6.1.0
     - ipykernel >=4.5.1
-    - traitlets >=4.3.1,<6.0.0
+    - traitlets >=4.3.1
     - nbformat >=4.2.0
-    - widgetsnbextension >=3.5.0,<3.6.0
-    - jupyterlab_widgets >=1.0.0
-    - ipython_genutils >=0.2.0,<0.3.0
+    - widgetsnbextension >=4.0.4,<5.0.0
+    - jupyterlab_widgets >=3.0.4,<4.0.0
 
 test:
   imports:
     - ipywidgets
   requires:
+    - jsonschema
     - pip
-    - python <3.10
+    - pytest-cov
```

Comment:
`pytest-cov` can be removed as it's not used by test commands.
Add missing `pytest`

Reply:
done